assets/js/index.js

```diff
@@ -35,6 +43,10 @@
         },
       },
       tags: true,
+    }).on('select2:select', function () {
+      form.off('keyup', '.select2-selection', _submitFormIfPressedEnter);
+    }).on('select2:opening', function () {
```

Comment:
I tried binding it to other events, to no success. Ideally, we won't have to wait for s2 to open so we could `submit()` the form, but if there's no clear way to do it otherwise, then I'm okay with this. I'd just disable the input upon submitting, imho it improves a bit the feedback you get from pressing enter.